### ISSUE TITLE
Improve how we communicate the privacy implications of running the performance tests

### ISSUE THREAD

**Issue by hellais** [ooni/probe#1074]
We cannot guarantee that users IP address is not collected as the test is reliant on MLab infrastructure.

Since the rollout of the new NDT backend MLab has started to include the IP address users inside of the test results, we should therefore improve how this is communicated to end users.

Currently the test card description says:

> Disclaimer: These tests rely on third party servers. We therefore cannot guarantee that your IP address will not be collected.

Which is a bit weak, we should look into making this copy more strong and also look into perhaps adding a modal in between the screens to get further informed consent when users trigger this test.

In the long term we should also look into convincing MLab to not collect IPs of OONI Probe tests when they have opted out from IP collection and maybe as a hot fix stripping the IP from the OONI Measurement (this does not prevent MLab from still collecting and publishing it).

**Comment by hellais** [comment]
cc @bassosimone

**Comment by hellais** [comment]
This might also be related to ooni/probe#1066

**Comment by agrabeli** [comment]
@hellais I've already edited the copy of the performance card as part of editing the copy to group the middlebox tests under the performance card. See: https://github.com/ooni/probe/issues/835#issuecomment-623416957

I have also edited the OONI Data Policy accordingly to reflect this: https://ooni.org/about/data-policy#data-we-collect

Is there some other place where we need to make these edits? If not, I'd suggest closing this ticket.
